strings/tips.txt

```diff
@@ -263,7 +264,12 @@ As a Devil, as long as you control at least one other soul, you will automatical
 At which time a Devil's nameth is spake on the tongue of man, the Devil may appeareth. 
 You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.
 When hacking doors, cutting and mending the "test light wire" will restore power to the door.
+When hacking, remote singulars act like a multitool's pluse when attached to a wire and pinged. This can allow you to hack things or set traps from far away.
 When crafting most items, you can either manually combine parts or use the crafting menu.
 Suit storage units not only remove blood and dirt from clothing, but also radiation!
 Remote devices will work when used through cameras. For example: Bluespace RPEDs and door remotes.
 Laser pointers can be upgraded by replacing its micro laser with a better one from RnD! Use a screwdriver on it to remove the old laser. Upgrading the laser pointer gives you better odds of stunning a cyborg, and even blinding people with sunglasses.
+Being out of combat mode makes you deal less damage to people, and objects when attacking.
+Resting makes you deal less damage when you attack people or objects.
+When not in combat mode you regenerate stamina faster, resting also helps this but not sleeping.
+When fighting NPCs or Xenomorphs remember to be in combat mode as you will take less damage from their attacks, and deal more damage.
```

Comment:
Remember to be in combat mode while in combat, as otherwise you will be penalized by taking more incoming damage and dealing less damage to your adversary.